pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -337,19 +343,17 @@ func (h *HandlerImpl) writeCheckpoint() error {
 	if err != nil {
 		return err
 	}
-	filepath := h.devicePluginManager.CheckpointFile()
-	return ioutil.WriteFile(filepath, dataJSON, 0644)
+	return h.store.Write(defaultCheckpoint, dataJSON)
 }
 
 // Reads device to container allocation information from disk, and populates
 // h.allocatedDevices accordingly.
 func (h *HandlerImpl) readCheckpoint() error {
-	filepath := h.devicePluginManager.CheckpointFile()
-	content, err := ioutil.ReadFile(filepath)
+	content, err := h.store.Read(defaultCheckpoint)
 	if err != nil && !os.IsNotExist(err) {
```

Comment:
IsNotExist() check is unnecessary, because it is checked already by the store.Read() function.